tests/php/ORM/Search/SearchContextTest.php

```diff
@@ -58,13 +58,24 @@ public function testSearchableFieldsDefaultsToSummaryFields()
         $obj = new SearchContextTest\NoSearchableFields();
         $summaryFields = $obj->summaryFields();
         $expected = [];
-        foreach ($summaryFields as $field => $label) {
+
+        $expectedSearchableFields = [
+            'Name',
+            'Customer.FirstName',
+            'Image',
+            'Image.BackLinks',
```

Comment:
Neither `Image` nor `Image.BackLinks` can be searched against, since they're not _fields_, they're a whole dataobject and a whole HasManyList resepectively.
```suggestion
```

Reply:
Updated